candle-onnx/src/eval.rs

```diff
@@ -766,6 +766,20 @@ pub fn simple_eval(
                 let output = input.cumsum(axis as usize)?;
                 values.insert(node.output[0].clone(), output);
             }
+            //  https://github.com/onnx/onnx/blob/main/docs/Operators.md#flatten
+            "Flatten" => {
+                let axis = get_attr_opt::<i64>(node, "axis")?.copied().unwrap_or(1) as usize;
+                let input = get(&node.input[0])?;
+                let end_index = input.shape().dims().iter().product::<usize>();
+                let new_shape = if axis == 0 {
+                    (1, end_index)
+                } else {
```

Comment:
What about the case where axis would not be 1?

Reply:
Hi， Referencing this link https://github.com/onnx/onnx/blob/main/docs/Operators.md#flatten, in the case where axis > 1, it is compatible with axis == 1.

Reply:
Couldn't the case with axis set to 0 be merged with the other case? The product from `first_part` would be `1` on an empty iterator?

Reply:
you are right. I made changes.